src/docs/asciidoc/api-guide/overview/terms-of-service.adoc

```diff
@@ -0,0 +1,18 @@
+[[terms-of-service]]
+== Terms of service
+
+=== Playback information
+As an API user, you are responsible for submitting video playback information to Boclips via our events API. If you are using Boclips' javascript player this will be done automatically, however, if you are not, you must submit these playback events yourselves. More information about how to do this can be found <<resources-events,`here`>>.
+
+=== Undocumented fields
+When using the API you may notice some fields in the response that are not documented. These undocumented fields are likely to change and must not be implemented against unless explicit approval has been given from Boclips (typically when testing beta features). The full extent of Boclips' API offering is explicitly documented in these docs.
+
+=== Changes to the API
+On the rare occasion that we need to make breaking changes to our API, the process of making this change will go like:
```

Comment:
Copy looks good 👍 

I'm unable to tell if below are bullet points or not. If they are not, I think it would be nice to lay them out in that way.

Reply:
The steps are numbered. This is what it looks like
![image](https://user-images.githubusercontent.com/9035723/88921708-e218e180-d266-11ea-8fa2-7428efbdd47a.png)